docs/workflow.md

```diff
@@ -0,0 +1,137 @@
+
+# Workflows
+
+Ths document documents two different yet common workflows in BKR using `kubeprod` and `kubecfg`:
+
+* Basic workflow
```

Comment:
I think we need some context here: who uses these workflows, what are the assumptions or knowledge needed, etc?. Also, I don't find "basic" and "advanced" labels to be descriptive enough, could we add some info on what is the purpose of each workflow or the end-user benefit of applying each workflow?

Reply:
PTAL